manifest.yml.j2

```diff
@@ -11,7 +11,7 @@ applications:
 
     processes:
     - type: web
-      command: celery -A run_celery.notify_celery worker --loglevel=INFO --concurrency=1
+      command: celery -A run_celery.notify_celery worker --loglevel=INFO --concurrency=1> /dev/null
```

Comment:
This looks slightly different to how to we it in other apps, where we do `2> /dev/null` e.g. https://github.com/alphagov/notifications-api/blob/6f9edf0c070466c72ea90bf6ad5878f9a2ef788d/scripts/paas_app_wrapper.sh#L9 - is that intentional?

Reply:
no I think I just copied it wrong - thanks